### ISSUE TITLE
Membership request std.uritemplate

### ISSUE THREAD

**Issue by baywet** [dotnet-foundation/projects#349]
## Your name: 

Vincent Biret (MSFT) @baywet
Andrea Peruffo (Red Hat) @AndreaTP

## Your Email: 

vincent.biret@microsoft.com
andrea.peruffo1982@gmail.com

## Your Discord ID: 

baywet
andreaTP

## Notes (OPTIONAL):

https://github.com/std-uritemplate/std-uritemplate/

As discussed this is an implementation of RFC 6570 across multiple languages. The main reason why we'd like to join the .NET foundation is so we can code-sign the dotnet package.
Right now this is setup as a mono-repo and we're evolving all languages as a common front (less management required this way).
To recap the earlier discussions there are mainly two options (assuming the project gets accepted):

- keep it as monorepo, configure the CLA bot to only enforce the /dotnet directory, make sure the agreement when joining the foundation is customized to reflect that. (preferred as it reduces maintenance burden, and avoids confusing contributors to other languages)
- set up a separate dotnet repo we'd (or not) sync back into this one.

**Comment by glennawatson** [comment]
I've added the appropriate permissions on discord for the project committee to both users.

**Comment by andreaTP** [comment]
Thanks @baywet for taking the time to submit this one!

> Andrea: what email did you want listed here?

my personal one, it's public on my GH profile.

**Comment by baywet** [comment]
@andreaTP Thanks, updated the initial issue.
@glennawatson do you need anything else from us? what are the next steps?

**Comment by baywet** [comment]
For context I also created a project application #353

**Comment by glennawatson** [comment]
Would you guys like to be members of the project committee?

Eg discuss project admissions/removals, policy decisions around projects, and maintainer support?

Just not sure because it seems your intent was #353?

**Comment by baywet** [comment]
No, we don't want to be in the committee.
We'd like for our project to join the foundation.
Sorry about the confusion, I thought membership was a pre-requisite to the project application.
